src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -183,4 +183,18 @@ static FabricLoader getInstance() {
 
 	@Deprecated
 	File getConfigDirectory();
+
+	/**
+	 * Gets the command line arguments used to launch the game. If this is printed for debugging, make sure {@code sanitize} is {@code true}.
+	 * @param sanitize Whether to remove sensitive information
+	 * @return the launch arguments
+	 */
+	String[] getLaunchArguments(boolean sanitize);
```

Comment:
Obviously anyone could call this with `false` as the parameter and run into the sunset with your access token.

Do we want to print a stacktrace when someone calls this so we can see who wanted to access the unsanitized arguments?

Reply:
is that really needed? I think it's just log spam.
Besides, it wouldn't be difficult for malicious mods to bypass this entire thing anyway.

Reply:
Well I do see your point there, it's probably a bit spammy tbf

Reply:
The arg is really just to remind people of the dangers involved, aiming at accidental leaks. We can't fundamentally avoid illegitimate disclosure, whether this exists or not, so it should be fine without extra logic that'd at most provide a false sense of security.